internal/host/plugin/job_set_sync.go

```diff
@@ -54,7 +54,7 @@ func newSetSyncJob(ctx context.Context, r db.Reader, w db.Writer, kms *kms.Kms,
 		return nil, errors.New(ctx, errors.InvalidParameter, op, "missing db.Writer")
 	case kms == nil:
 		return nil, errors.New(ctx, errors.InvalidParameter, op, "missing kms")
-	case len(plgm) == 0:
+	case plgm == nil:
```

Comment:
Is this change correct? Is an empty map ok?

Reply:
Yeah -- I checked this with Todd. The intent of the check is to ensure we don't access a nil pointer; if there are no plugins then it shouldn't fail, it should have nothing to do.